README.MD

```diff
@@ -0,0 +1,43 @@
+# Software Design - Supply Chain
+An implementation of "Supply Chain" using Spring Boot framework
+
+## Build instructions
+```
+git clone https://github.com/kaca1707/supplychain.git
+cd supplychain
+```
+
+On Linux:
+```
+./mvnw spring-boot:run
+```
+
+On Windows:
+```
+./mvnw.cmd spring-boot:run
+```
+
+Go to your webbrowser and type:
+```
+http://localhost:8080/
+```
+
+## Running in IntelliJ
+Go to File -> New -> Project from Version Control -> Git -> Paste URL to Git Repo
```

Comment:
If they already have pulled the repository, the can just open the project.

Reply:
Yeah, but i think that this can be usefull for those who dont want to use command line and are not familiar with Intelli J